ui/src/app/pages/admin/admin-user-bypass.tsx

```diff
@@ -50,6 +50,7 @@ const getBypassedModules = (user: AdminTableUser): Array<AccessModule> => {
     ...(user.dataUseAgreementBypassTime ? [AccessModule.DATAUSEAGREEMENT] : []),
     ...(user.eraCommonsBypassTime ? [AccessModule.ERACOMMONS] : []),
     ...(user.twoFactorAuthBypassTime ? [AccessModule.TWOFACTORAUTH] : []),
+    ...(user.rasLinkLoginGovBypassTime ? [AccessModule.RASLINKLOGINGOV] : []),
```

Comment:
You can also conditionally add this to the registration dashboard test-only "bypass all" flow: https://github.com/all-of-us/workbench/blob/81a6213a125eee09ea537ecbb78dfe74961a6bcc/ui/src/app/pages/homepage/registration-dashboard.tsx#L316